examples/launch_height_benchmark_remotely.py

```diff
@@ -0,0 +1,48 @@
+# Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+"""
+This example show how to launch a tuning job that will be executed on Sagemaker rather than on your local machine.
+"""
+import logging
+from pathlib import Path
+
+from benchmarking.commons.launch_remote_sagemaker import launch_remote
```

Comment:
Should be `launch_remote_local`, not `launch_remote_sagemaker`.

I think you are mixing up SageMaker backend with remote launching. The namings `local`, `sagemaker`, `simulator` always refer to the backend.

So if you launch experiments with local backend remotely (maybe the most common use case), you need to use `hpo_main_local` and `launch_remote_local`.

Reply:
I think there is some misunderstanding here. This example launches jobs remotely using sagemaker backend which is why I am using launch_remote_sagemaker.